src/components/SentimentAnalysisGraph.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { VictoryAxis } from 'victory';
+import { VictoryBar } from 'victory';
+import { VictoryChart } from 'victory';
+
+class SentimentAnalysisGraph extends Component {
+	static displayName = 'SentimentAnalysisGraph';
+
+	state = {
+		callFunc: this.getData(),
+		data: {pos_value: 90, neg_value: 10}
+	};
```

Comment:
You can remove state from this component.
`callFunc` looks like it is never used, you can get rid of it
`data` should be a prop, add a `static defaultProps = {...}` for 90/10
Change the names of `pos_value` and `neg_value` (variable names should be camelCase)